lib/model/model.go

```diff
@@ -2011,6 +2011,7 @@ func (s *indexSender) sendIndexTo(ctx context.Context) error {
 					l.Warnln("Failed repairing sequence entries:", dbErr)
 					panic("Failed repairing sequence entries")
 				} else {
+					s.evLogger.Log(events.Failure, "detected and repaired non-increasing sequence")
```

Comment:
Still at a loss how knowing this is useful?

I guess to verify when the bug has been fixed?

Reply:
Yep. We have no clue how bad the problem is right now, because we have an ugly mitigation. Lately there is the "device missing but present in global" thing that with actual db error handling might be mitigatable too, but I don't want to just mitigate without any reporting. Without reporting if a fix comes in (as it hopefully lately did for this exact problem), we have no clue whether the fix was effective or not.